packages/portal-proto/src/pages/404.tsx

```diff
@@ -1,10 +1,15 @@
 import { NextPage } from "next";
 import Head from "next/head";
+import { datadogRum } from "@datadog/browser-rum";
 import { UserFlowVariedPages } from "@/features/layout/UserFlowVariedPages";
 import { headerElements } from "@/features/user-flow/workflow/navigation-utils";
 import Link from "next/link";
 
 const Custom404Page: NextPage = () => {
+  datadogRum.startView({
+    name: "Not Found",
+  });
```

Comment:
Do you know how this behaves if we do not initialize datadog?  I'm hoping it silently does nothing as opposed to throwing an error.

Reply:
Yeah, I've tested it without initializing datadog and it is just ignored.